src/views/set2/Set2.vue

```diff
@@ -13,21 +13,20 @@
         >
           <br>
           <p class="narrative_text">
-            The USGS and
-            state and local agencies monitor stream temperature across the Delaware
-            River Basin, collecting continuous and discrete measurements of water temperature.
-            In the past 40 years, stream temperature has been measured at more than
-            1,300 sites throughout the basin. Expansion to the monitoring network since 2018
-            is rapidly increasing the volume of available data.
+            At the surface, it appears we have a lot of temperature information in the basin. 
```

Comment:
ticky tacky, but I'd prefer we not use "at the surface" here, since it has a dual meaning. But I'm also assuming I'm not reviewing final-final text, so not very important to make a change here, just noting it as a potential source of confusion.

Reply:
Good point - maybe 'at first glance' would be better. Also, yes, this is still definitely draft text. It will be tweaked further once layout is finalized and ch. 3 components are further along.